clog/loggers.py

```diff
@@ -239,6 +242,17 @@ def close(self):
         self.connected = False
 
 
+class MonkLogger(object):
+    """Wrapper around MonkProducer"""
+
+    def __init__(self, client_id, host=None, port=None):
+        self.producer = MonkProducer(client_id, host, port)
+
+    def log_line(self, stream, line):
+        clog_stream = '_clog.{0}'.format(stream)
```

Comment:
I would rather not to do this. I think we can configure streams without using an hardcoded prefix. If we really want a prefix we can add a temporary config option for it so that we can control it without code changes.